arduino/TFT_Matrix/TFT_Matrix.ino

```diff
@@ -3,6 +3,9 @@
 // then scroll smoothly
 
 // Needs GLCD font
+// adapted for 135x240 screen from TTGO T-Display
+
+// Here https://github.com/Bodmer/TFT_eSPI/issues/493 the comment ghostoy solved the problem
```

Comment:
Even though the ST7789 supports higher resolutions, this one is for T-Display with just 135x240 pixels.